### ISSUE TITLE
Feature. add --cookie to use browser cookies for comics that require a log in

### ISSUE THREAD

**Issue by KeePach** [J-CPelletier/webcomix#91]
I've found this software to be incredible and easy to use and I love it, I ask this because there are some comics that require you to be logged in in a website in order the view the pages and webcomix can't download the comic for this reason. I was thinking that maybe using the browser cookies of the logged in website would solve this problem.

**Comment by J-CPelletier** [comment]
This could be done, although it would also depend on the kind of cookies, since some might not easily be changed since session tokens are handled differently by every website. I could potentially check if we can add a specific cookie name/value entry to handle login if possible. Do you have an example of the kind of site that would require log-in?

**Comment by KeePach** [comment]
yes, for example https://www.theduckwebcomics.com/ is such a site, some comics can't be accessed if you're not logged in.

**Comment by J-CPelletier** [comment]
I just a new feature that allows people to pass cookies by adding `--cookie <key> <value>` to a `webcomix custom` command with webcomix `3.11.0`.
